src/Sylius/Behat/Context/Setup/CartContext.php

```diff
@@ -46,4 +48,47 @@ public function iAddedProductToTheCart(ProductInterface $product, string $tokenV
             1
         ));
     }
+
+    /**
+     * @Given /^I have (product "[^"]+") with (product option "[^"]+") ([^"]+) in the (cart)$/
+     */
+    public function iAddThisProductWithToTheCart(
+        ProductInterface $product,
+        ProductOptionInterface $productOption,
+        string $productOptionValue,
+        string $tokenValue
+    ): void {
+        $this->commandBus->dispatch(AddItemToCart::createFromData(
+            $tokenValue,
+            $product->getCode(),
+            $this
+                ->getProductVariantWithProductOptionAndProductOptionValue(
+                    $product,
+                    $productOption,
+                    $productOptionValue
+                )
+                ->getCode(),
+            1
+        ));
+    }
+
+    private function getProductVariantWithProductOptionAndProductOptionValue(
+        ProductInterface $product,
+        ProductOptionInterface $productOption,
+        string $productOptionValue
+    ): ?ProductVariantInterface {
+        foreach ($product->getVariants() as $productVariant) {
+            /** @var ProductOptionInterface $variantProductOption */
+            foreach ($productVariant->getOptionValues() as $variantProductOption) {
```

Comment:
Shouldn't it be:
```suggestion
            /** @var ProductOptionValueInterface $variantProductOptionValue */
            foreach ($productVariant->getOptionValues() as $variantProductOptionValue) {
```
?